src/__tests__/index.getFirebaseAdmin.server.test.js

```diff
@@ -2,6 +2,9 @@
  * @jest-environment node
  */
 
+// Allow importing firebase-admin as wildcard.
+/* eslint-disable no-import-assign */
```

Comment:
Not sure it is helpful since you are still using the old import style, but I've personally worked with the new version of firebase-admin's imports with eslint-plugin-import by doing the following within my eslint rules:

```js
    // NOTE: This is added since eslint-plugin-import does not support exports in package.json
    // which is what firebase-admin v10 uses. See: https://github.com/import-js/eslint-plugin-import/issues/1810
    'import/no-unresolved': [
      2,
      {
        ignore: [
          'firebase-admin/database',
          'firebase-admin/app',
          'firebase-admin/auth',
          'firebase-admin/storage',
          'firebase-admin/firestore',
        ],
      },
    ],
```

Reply:
Thanks, I'll plan to incorporate this later when we drop support for Firebase Admin v9.